src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,12 @@
 
 public class Application {
     public static void main(String[] args) {
+        Lottery lottery = new Lottery();
+        ColorSupplier supplier = new ColorSupplier();
+        for (int i = 0; i < 3; i++) {
```

Comment:
Instead of using magic numbers directly in your code, such as the number '3' for the loop iteration, consider defining it as a constant at the beginning of your class. This improves code readability and maintainability.